### PR TITLE
Flash

### PR DESCRIPTION
**Pull request by jesswray**
More of the same.  Don't mind the commit that says "everything's broken". That was just the attachinary merge

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -9,10 +9,25 @@ def create
       redirect_to prompts_path, notice: "You have created yo' self!"
       session[:user_id] = @user.id
     else
+      if @user.errors.messages == {:password=>["can't be blank"]}
```
Comment (notnotdrew) [inline app/controllers/users_controller.rb:13]: There may be a way to create custom messages for errors? No big deal though, this gets it done.

**Comment by notnotdrew** [comment]
:+1:

**Comment by rolandobrown** [comment]
:+1:

**Comment by stefania11** [comment]
:+1:

**Pull request merged**